### PR TITLE
Add blacken-docs to pre-commit.

### PR DESCRIPTION
**Pull request by MaxJPRey**
`blacken-docs` tool will check the style of the code samples in the documentation.
In addition update hooks version.

### CODE REVIEW

File: doc/source/api/evaluate/index.rst [hunk 1]
```diff
@@ -16,15 +16,15 @@ to facilitate the manipulation and execution of Twin Runtimes through TwinModel.
 Workflow Example
 ----------------
 
-.. code-block:: python
+.. code-block:: pycon
```
Comment (lboucin) [inline doc/source/api/evaluate/index.rst:19]: typo: python (or maybe not since it is used in other files?)

File: doc/source/user_guide/index.rst [hunk 2]
```diff
@@ -13,22 +13,26 @@ You can change these global settings at anytime using the following functions:
 
 .. code-block:: python
```
Comment (lboucin) [inline doc/source/user_guide/index.rst:14]: pycon or python?

File: doc/source/api/sdk/index.rst [hunk 3]
```diff
@@ -16,13 +16,13 @@ Twin Runtime SDK functionalities.
 Workflow Example
 ----------------
 
-.. code-block:: python
+.. code-block:: pycon
```
Comment (lboucin) [inline doc/source/api/sdk/index.rst:19]: pycon or python?

**Review by lboucin — APPROVED** [review]
Approved with suggestions (see comments).

File: doc/source/api/evaluate/index.rst (hunk 1 above)
Comment (MaxJPRey) [reply doc/source/api/evaluate/index.rst:19]: @lboucin It is `pycon`. Actually with the new check of the documentation, pre-commit was failling.
[pycon](https://pvbookmarks.readthedocs.io/en/master/documentation/doc_generators/sphinx/rest_sphinx/code_block.html#pycon-python-console-highlighter) is for interactive Python highlighter.

File: doc/source/user_guide/index.rst (hunk 2 above)
Comment (lboucin) [reply doc/source/user_guide/index.rst:14]: @MaxJPRey So I guess we should update this one too? Shouldn't we?

File: doc/source/api/evaluate/index.rst (hunk 1 above)
Comment (lboucin) [reply doc/source/api/evaluate/index.rst:19]: Ok thanks!

File: doc/source/user_guide/index.rst (hunk 2 above)
Comment (MaxJPRey) [reply doc/source/user_guide/index.rst:14]: No this one is OK, because it is pure Python code, without the `>>>` in front of each line.

Comment (MaxJPRey) [reply doc/source/user_guide/index.rst:14]: I could do that because we are not displaying any output from a `print`.

Comment (lboucin) [reply doc/source/user_guide/index.rst:14]: @MaxJPRey Ok thanks a lot for the clear explanation

**Pull request merged**